### DISCUSSION TITLE
[help]

### DISCUSSION THREAD

**Discussion by zhaozhijun1988** [Chia-Network/chia-blockchain#3336] [Support]
how to get coin id from rpc?
parent_coin_info is the parent coin id? coinid = sha256(parent_coin_info, puzzle_hash, value),how to get coin id by rpc interface, thanks

**Comment by sargonas** [comment]
Migrating this issue to the Support section in the Discussion forums. You will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

**Reply by PrernaPadhee** [reply]
hey, are there any resolutions on this ?

**Reply by sargonas** [reply]
I believe this is covered in our developer documentation at https://docs.chia.net.

If not, feel free to swing through our developer forums at https://developers.chia.net where our Dev Relations team can assist you (and also use that as an opportunity to update the documentation if it is missing.)

**Reply by PrernaPadhee** [reply]
I am making use of c# to consume the data from my chia node. I have used the following function to get the coin_id, however the actual response doesnt match with the expected value. Can you please provide me with a solution in c# for the same?

private static String sha256_hash(string value)
        {
            using var hash = SHA256.Create();
            var byteArray = hash.ComputeHash(Encoding.UTF8.GetBytes(value));
            return Convert.ToHexString(byteArray).ToLower();
        }


string coin_id = sha256_hash(parent_coin_id + puzzle_hash + amount.ToString());

**Reply by sargonas** [reply]
The best place for this kind of discussion is our developer forums at https://developers.chia.net/, they can definitely get you sorted there!

**Reply by PrernaPadhee** [reply]
Ok, I will check there. Thanks!
